Content/arm-template.json

```diff
@@ -7,9 +7,11 @@
         "pricingTier": { "type": "string" }
     },
     "variables": {
-        "prefix": "[concat('safe-', parameters('environment'))]",
+        "environment": "[toLower(parameters('environment'))]",
+        "prefix": "[concat('safe-', variables('environment'))]",
         "appServicePlan": "[concat(variables('prefix'), '-web-host')]",
         "web": "[concat(variables('prefix'), '-web')]",
+        "storage": "[concat('safe', variables('environment'), 'storage')]",
```

Comment:
why don't use same naming convention (add dashes) as for other resources?

Reply:
Storage accounts have stricter requirements. No dashes, max of 24 characters and all lower case. In fact I'll probably add some more code into the template or script to enforce them all.